docs/adguard-vpn-for-linux/settings.md

```diff
@@ -52,14 +99,36 @@ Replace `<server_address>` with the address of your DNS server. To use this DNS
 
 ## VPN tunnel routing mode: NONE, AUTO, or SCRIPT
 
-You can choose how AdGuard VPN routes traffic through the VPN tunnel. To set the tunnel routing mode to NONE (no routing), type:
+You can choose how AdGuard VPN CLI routes traffic through the VPN tunnel. Routing mode controls system routing when the TUN interface is active. It does not auto-redirect traffic in SOCKS5/proxy mode: apps must be configured to use the local SOCKS5 proxy in that case.
+
+### NONE — no routing changes
+
+AdGuard VPN CLI brings the TUN interface up but does not modify the system routing table. No default route to the tunnel is installed — traffic continues to use existing OS routes.
+
+Use this if you want to manage routes yourself (manually or with third-party tools), or when you only use the SOCKS5 proxy mode and do not need the system to push traffic into the TUN.
```

Comment:
> when you only use the SOCKS5 proxy mode and do not need the system to push traffic into the TUN.

It sounds like this setting has some meaning in SOCKS5 mode, while this setting does not relate to it.

I'd prefer to not to mention it.